build.sbt

```diff
@@ -330,30 +332,11 @@ lazy val docSettings = Seq(
   micrositeDocumentationUrl := "https://www.javadoc.io/doc/org.typelevel/spire_2.13/latest/spire/index.html",
   micrositeDocumentationLabelDescription := "API Documentation",
   micrositeExtraMdFiles := Map(
-    file("AUTHORS.md") -> ExtraMdFileConfig(
-      "authors.md",
-      "home",
-      Map("title" -> "Authors", "section" -> "Home", "position" -> "5")
-    ),
-    file("CHANGES.md") -> ExtraMdFileConfig(
-      "changes.md",
-      "home",
-      Map("title" -> "Changes", "section" -> "Home", "position" -> "2")
-    ),
```

Comment:
I think it's helpful to continue to keep these at the top of the repo. For example cats is like this:
https://github.com/typelevel/cats

Reply:
Ah, I moved them when I tried to do this hack https://github.com/typelevel/spire/pull/1095#issuecomment-950253326

I need move them back.

Reply:
Btw, I think `DESIGN.md` an `FRIENDS.md` will be likely to contain code block(e.g. for explanation or example). Do you think they should be moved back too?

Reply:
(Probably you mentioned line 333 to 342 only. Just to be sure.)

Reply:
Hmm, good question ... you are right `DESIGN.md` could have code blocks, but currently they do not I think.

Would it be okay with you, if we put it back?
1. Cats also has `DESIGN.md` in its top directory: https://github.com/typelevel/cats
2. I'm conservative about making changes here, since I am not an original author of the library, sorry :(

Reply:
No worries!